README.rst

```diff
@@ -36,3 +36,42 @@ Run tests:
 
   $ cd bluesky-kafka
   $ pytest
+
+Run a Mongo Consumer Group
+--------------------------
+
+Create a conda environment:
+
+::
+
+  $ conda create -n consumers python=3.8
+  $ conda activate consumers
+
+Install packages:
+
+::
+
+  $ pip install bluesky-kafka supervisor
+
+Setup environment variables:
+mongo_uri reference: https://docs.mongodb.com/manual/reference/connection-string/
+bootstrap_servers: comma-separated list of brokers.
+
+::
+
+  $ export BLUESKY_MONGO_URI="mongodb://username:password@machine1:port1,machine2:port2,machine3:port3
+  $ export KAFKA_BOOTSTRAP_SERVERS="machine1:9092, machine2:9092, machine3:9092"
```

Comment:
Is it ok to have spaces in the list of bootstrap servers?

Reply:
Spaces work. Does it work without spaces? Maybe I'll get rid of the spaces so it matches the mongo uri